client/src/components/surveys/SurveyNew.js

```diff
@@ -0,0 +1,34 @@
+// SurveyNew shows the SurveyForm and SurveyFormPreview
+import React, { Component } from 'react';
+import { reduxForm } from 'redux-form';
+import SurveyForm from './SurveyForm';
+import SurveyFormReview from './SurveyFormReview';
+
+class SurveyNew extends Component {
+  //state initialization
+  state = { showFormReview: false };
+
+  renderContent() {
+    if (this.state.showFormReview) {
+      return (
+        <SurveyFormReview
+          onCancel={() => this.setState({ showFormReview: false })}
+        />
+      );
+    }
+
+    return (
+      <SurveyForm
+        onSurveySubmit={() => this.setState({ showFormReview: true })}
+      />
+    );
+  }
+
+  render() {
+    return <div>{this.renderContent()}</div>;
```

Comment:
Think this is overkill in render I would just render the above SurveyForm

Reply:
thank you! I'll come back to this... I'll put it in my TODO's for refactor..